data/pb/ws-services-calculation/meterStatus.json

```diff
@@ -0,0 +1,36 @@
+{
+  "tenantId": "pb",
+  "moduleName": "ws-services-calculation",
+  "meterStatus": [
```

Comment:
Could you make the master names sentence case? MeterStatus ?

Reply:
ok